### DISCUSSION TITLE
Where is the value in using this?

### DISCUSSION THREAD

**Discussion by MSBassSinger** [microsoft/aspire#4356] [General]
I walked through the Aspire .NET Starter app in VS 2022.  It doesn't even start without an error.

_Container runtime 'docker' was found but appears to be unhealthy. The error from the container runtime check was error during connect: in the default daemon configuration on Windows, the docker client must be run with elevated privileges to connect: Get "http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.24/containers/json?limit=1": open //./pipe/docker_engine: The system cannot find the file specified._

If this is supposed to be "cloud native", then why the containers and kubernetes?  Azure cloud native does not need developers to waste time on those cloud-hosted tools.  Whether MS uses them "under the covers" for Azure cloud native doesn't matter.  What matters is where Azure provides scalability as promised.

And why use Entity Framework?  That monstrosity only slows down an app, increases complexity unnecessarily, and does not scale weel, particularly as an app matures.  I've seen several software shops take EF out of their .NET apps after two or three years of trying to make it work.

So, from a value engineering perspective, what value is offered with this hodgepodge referred to as Aspire, as compared to simply building a cloud app with better, leaner, less complex, more scalable, cloud native architecture?  And Microsoft making Aspire opinionated (which is ironic since MS has made a practice of non-opinionated development tools) doesn't seem like a plus.  

At least make your sample app run right the first time.

**Comment by davidfowl** [comment]
If you run docker desktop it should start up the sample.

**Reply by danmoseley** [reply]
@MSBassSinger How can we make the error message more helpful?

**Reply by davidfowl** [reply]
There's a work item to allow the IDE to do this by default but we haven't yet implemented the the error codes to make this happen. 

@danegsta @BillHiebert
